app/user/user.js

```diff
@@ -79,6 +79,16 @@ User.prototype.removeInvite = function removeInvite(inviteKey) {
     });
 };
 
+User.prototype.removeInstitution = function removeInstitution(institutionKey) {
```

Comment:
esse remove pode dar erro se só tiver 1 inst e esta for removida, quando for fazer pegar o índice 0 da lista vai dar undefined, e a current_institution vai ficar bugada.

Reply:
antes de bugar ele sai da aplicação porque o state do user fica inactive, não?